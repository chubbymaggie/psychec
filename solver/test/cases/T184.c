void f()
{
    const char* name;
    if (*name)
    {
        quote (name);
        quote ("foo");
    }
}

void f2()
{
    char* name2;
    if (*name2)
    {
        quote2 (name2);
        quote2 ("foo");
    }
}
