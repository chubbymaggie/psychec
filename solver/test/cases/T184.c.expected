/* Forward declarations */

/* Type definitions */

/* Variables and functions */
 int /*<<< orphan*/  quote (char const*) ; 
 int /*<<< orphan*/  quote2 (char*) ; 

